pages/patientProjects.js

```diff
@@ -0,0 +1,27 @@
+export const projects = [

```

Comment:
This should maybe be the default export, and this file could live in, say, `/data/`?

Reply:
Ah, this actually _needs_ to be out of `/pages/` – the build is failing because of that.